integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakTestResource.java

```diff
@@ -54,6 +57,21 @@
                                         "  ]\n" +
                                         "}")));
 
+        // define the mock for the introspect endpoint
+        WireMock.stubFor(WireMock.post("/auth/realms/quarkus/protocol/openid-connect/token/introspect")
```

Comment:
It is not in the header, but in the POST payload

Reply:
My bad. Converted to body payload.